ONSdigital/census-rm-sample-loader
# Check for whitespace in CSV sample file

# Motivation and Context
DR has noticed that some fields in sample files should be blank but actually contain whitespace which could cause issues further down the line if recognised as a value

# What has changed
Added in a new validator to check if field starts with a whitespace
Created unit test for happy and unhappy path

# How to test?
Edit a test sample in the repo by adding in blank spaces where the field is expected to be blank
Then run `pipenv run python validate_sample.py <sample_file>`
Validation error should come up specifying which row/column

# Links
https://trello.com/c/yUU8RM7J/830-add-white-space-check-to-sample-load-validator-3
